project.v4.0.md

```diff
@@ -0,0 +1,787 @@
+- **Linkability (LINDDUN):** Pseudonyms are unlinkable in design, but repeated behavior or metadata correlation (timing, phrasing, stylometry) could deanonymize dissidents.  
+- **Tampering:** Trust-graph attestations are local first; delayed anchoring leaves a window for manipulation before snapshot proofs.
+
+### 2. Discussion & Deliberation
+- **Flooding / DoS:** Open threads invite brigading by extremists, trolls, or authoritarian states. Trust weighting can demote, but only after damage/noise occurs.  
+- **Bias Amplification:** NLP/LLM tools used for clustering and tone-flagging can be adversarially manipulated (prompt injection, adversarial text) to distort consensus maps.  
+- **Detectability:** Even with pseudonyms, activity patterns can reveal that a given pseudonym participated in a controversial thread. If an individual reuses the same pseudonym across all aspects of their expression, this can become incriminating under repressive regimes.
+
+### 3. Voting & Polling
+- **Eligibility Proofs:** Commit–reveal + ZK is robust, but implementation errors could leak side-channel info.  
+- **Vote Coercion:** Users may be forced to reveal their commit nonce to prove how they voted — a classic coercion risk.  
+- **Information Disclosure:** Proof-free polls, if aggregated without sufficient differential privacy, risk revealing minority positions (e.g., small LGBTQ+ groups).
+
+### 4. Organizational Governance
+- **Takeover Risks:** Membership APIs are intentionally flexible, but hostile actors (corporations, authoritarian governments) could define rules that look open while covertly excluding or deanonymizing.  
+- **Repudiation:** Policy changes may not leave tamper-proof audit trails if governance metadata is not consistently anchored.
+
+### 5. Analytics & Feedback
+- **Differential Privacy Weaknesses:** Poorly tuned ε (epsilon) values can leak sensitive micro-group data.  
+- **Re-identification:** Cross-correlating ideological heatmaps with external datasets could expose dissidents, whistleblowers, or minorities.  
+- **Bias & Manipulation:** Automated summarization could misrepresent minority opinions, shaping perception.
+
+### 6. Network & Privacy Layer
+- **Metadata Leaks:** Even Tor/I2P/mixnets struggle against global passive adversaries. Timing analysis may still correlate activity with individuals.  
+- **Denial of Service:** Federated nodes are choke points; targeted DoS or resource exhaustion could silence whole collectives.  
+- **Elevation of Privilege:** Node operators may exploit admin consoles for surveillance beyond intended scope.
+
+### 7. Supply Chain & Updates
+- **Tampering / Malicious Updates:** If release signing or reproducible builds are not rigorously enforced, adversaries can push backdoored clients.  
+- **Rollback Attacks:** Users may be tricked into older, vulnerable versions without strong anti-rollback enforcement.
```

Comment:
Firmware level backdoors are a very real threat. State actors have been known to put malicious code onto HDD firmware (circa 2015). Finding that date proved there were other firmware level threats before and after. Starting in 2008 it seems.

This section needs to be updated to reflect this threat.